### PR TITLE
Fixed a few spelling mistakes

### PR DESCRIPTION
**Pull request by ColinEberhardt**
Thanks to @lukeapage 

```
npm i markdown-spellcheck -g
mdspell -axn "site/src/**/*.md"
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,6 @@
 doc
 dist
 node_modules
-.*
```
Comment (chrisprice) [inline .gitignore]: I would prefer we left this in and force added the relevant files.

File: .spelling [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+programmatically
+datetime
+D3
+crosshair
+trackingstart
+trackingmove
+trackingend
+crosshairs
+ - site/src/components/indicator/relativeStrengthIndicator.md
+14-day
```
Comment (chrisprice) [inline .spelling]: Missing newline

File: site/src/components/introduction/2-decorate-pattern.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ Most chart APIs are complex and expansive in order to provide flexibility. With
...
 
-Unfortunately the [D3 component pattern](http://bost.ocks.org/mike/chart/), where re-useable units are encapsulated as functions, also encapsulates the data-join, hiding its power. For example, the [D3 axis component](https://github.com/mbostock/d3/wiki/SVG-Axes) uses a data-join internally, but as a consumer of this component you cannot add extra logic to the enter selection. The decorate pattern addresses this issue, by allowing the construction of components in such a way that their internal data join is 'exposed'.
+Unfortunately the [D3 component pattern](http://bost.ocks.org/mike/chart/), where reusable units are encapsulated as functions, also encapsulates the data-join, hiding its power. For example, the [D3 axis component](https://github.com/mbostock/d3/wiki/SVG-Axes) uses a data-join internally, but as a consumer of this component you cannot add extra logic to the enter selection. The decorate pattern addresses this issue, by allowing the construction of components in such a way that their internal data join is 'exposed'.
```
Comment (chrisprice) [inline site/src/components/introduction/2-decorate-pattern.md]: What's the change?

**Comment by chrisprice** [comment]
Very nice!

File: .gitignore (hunk 1 above)
Comment (ColinEberhardt) [inline .gitignore]: > I would prefer we left this in and force added the relevant files.

done.

File: site/src/components/introduction/2-decorate-pattern.md (hunk 3 above)
Comment (ColinEberhardt) [inline site/src/components/introduction/2-decorate-pattern.md]: re-usable => reusable

Comment (lukeapage) [inline site/src/components/introduction/2-decorate-pattern.md]: annoyingly once a line gets long github seems to not highlight the word changes..

File: .spelling (hunk 2 above)
Comment (lukeapage) [inline .spelling]: https://github.com/lukeapage/node-markdown-spellcheck/issues/23

**Pull request merged**